tests/unit/amo/components/Errors/TestServerError.js

```diff
@@ -30,6 +31,9 @@ describe(__filename, () => {
   it('renders a server error', () => {
     const root = render();
 
+    expect(root.find(ErrorComponent)).toHaveProp('code', 500);
+    expect(root).toHaveProp('header', 'Server Error');
```

Comment:
This should be `expect(root.find(ErrorComponent)).toHaveProp('header', 'Server Error');`